### ISSUE TITLE
Command Interface crash after rebinding it to primary mouse button

### ISSUE THREAD

**Issue by deathhimself23** [Regalis11/Barotrauma#5565]
- [x] I have searched the issue tracker to check if the issue has already been reported.

**Description**
game crashed when i opened the command list and pressed movement

**Steps To Reproduce**
i used keybinding leftmouse to use the command and pressed movement

**Additional information**

```
Barotrauma Client crash report (generated on 2021-04-22 1:30:30 PM)


Barotrauma seems to have crashed. Sorry for the inconvenience! 


97A4E8836E45EFAA8523C71EE09AA112


Game version 0.13.0.11 (ReleaseWindows, branch release, revision 840b932557)
Graphics mode: 1920x1080 (BorderlessWindowed)
VSync ON
Language: English
Selected content packages: Vanilla 0.9, Miroung
Level seed: nLoZLLtza
Loaded submarine: Dugong_Tutorial (8C7AD2F97BC9E420E76518CB2FE7E33D)
Selected screen: Barotrauma.GameScreen
SteamManager initialized


System info:
    Operating system: Microsoft Windows NT 10.0.19043.0 64 bit
    GPU name: AMD Radeon(TM) Graphics
    Display mode: {Width:1920 Height:1080 Format:Color AspectRatio:1.7777778}
    GPU status: Normal


Exception: Object reference not set to an instance of an object. (System.NullReferenceException)
Target site: Void SetReturnNode(Barotrauma.GUIButton, Microsoft.Xna.Framework.Point)
Stack trace: 
   at Barotrauma.CrewManager.SetReturnNode(GUIButton node, Point offset) in <DEV>\Barotrauma\BarotraumaClient\ClientSource\GameSession\CrewManager.cs:line 2023
   at Barotrauma.CrewManager.NavigateForward(GUIButton node, Object userData) in <DEV>\Barotrauma\BarotraumaClient\ClientSource\GameSession\CrewManager.cs:line 1940
   at Barotrauma.GUIButton.Update(Single deltaTime) in <DEV>\Barotrauma\BarotraumaClient\ClientSource\GUI\GUIButton.cs:line 262
   at System.Collections.Generic.List`1.ForEach(Action`1 action)
   at Barotrauma.GUI.Update(Single deltaTime) in <DEV>\Barotrauma\BarotraumaClient\ClientSource\GUI\GUI.cs:line 1232
   at Barotrauma.GameMain.Update(GameTime gameTime) in <DEV>\Barotrauma\BarotraumaClient\ClientSource\GameMain.cs:line 974
   at Microsoft.Xna.Framework.Game.DoUpdate(GameTime gameTime) in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\Game.cs:line 656
   at Microsoft.Xna.Framework.Game.Tick() in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\Game.cs:line 500
   at Microsoft.Xna.Framework.SdlGamePlatform.RunLoop() in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\SDL\SDLGamePlatform.cs:line 92
   at Microsoft.Xna.Framework.Game.Run(GameRunBehavior runBehavior) in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\Game.cs:line 397
   at Microsoft.Xna.Framework.Game.Run() in <DEV>\Libraries\MonoGame.Framework\Src\MonoGame.Framework\Game.cs:line 367
   at Barotrauma.Program.Main(String[] args) in <DEV>\Barotrauma\BarotraumaClient\ClientSource\Program.cs:line 58
```

**Comment by Rokvach** [comment]
I was able to reproduce this, just change the keybind for the command interface to be "left mouse" and click any of the commands

**Comment by ekqvist** [comment]
Fixed in Regalis11/Barotrauma-development@e51b3a0. While fixing this, I also noticed that you can only use the interface in the "clickless selection mode" (holding the Command Interface key) if you have rebound the Command Interface key to the same key as Deselect. Otherwise, opening the interface will also immediately close it. Not necessarily as a big of a deal, but something to maybe look into anyway.

**Comment by Rokvach** [comment]
Tested, wasn't able to reproduce anymore. Working correctly, closing.
